grpc/bcodes.go

```diff
@@ -37,3 +53,57 @@ func ErrorToProb(err error) *probs.ProblemDetails {
 		Detail: grpc.ErrorDesc(err),
 	}
 }
+
+func errorToCode(err error) codes.Code {
```

Comment:
Do you think its overkill to have a couple of unit tests for `errorToCode` and `unwrapError`? i'm on the fence.